nemo/core/classes/exportable.py

```diff
@@ -134,7 +134,7 @@ def export(
         dynamic_axes=None,
         check_tolerance=0.01,
     ):
-        my_args = locals()
+        my_args = locals().copy()
```

Comment:
Shouldn't this be a deep copy ?

Reply:
No its just to shut top warnings about locals list itself

Reply:
Ah ok